### PR TITLE
Do not run the consolidated pipeline when changing gitlab yml file

### PR DESCRIPTION
**Pull request by pierotibou**
Fixes #

Changes proposed in this pull request:

@DataDog/apm-dotnet

### CODE REVIEW

File: .azure-pipelines/ultimate-pipeline.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ trigger:
       - LICENSE
       - LICENSE-3rdparty.csv
       - NOTICE
+      - .gitlab-ci.yml
```
Comment (andrewlock) [inline .azure-pipelines/ultimate-pipeline.yml:19]: I think it shouldn't be here (in the `master` exclude list), it should be in the PR exclude list. As for the fact that there are 2 very different lists... yeah...

Comment (pierotibou) [reply .azure-pipelines/ultimate-pipeline.yml]: Yes I've done that way to quickly indeed. Thanks

**Comment by andrewlock** [comment]
## Benchmarks Report :snail:

Benchmarks for #2347 compared to [master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022): 
* 1 benchmarks are **faster**, with geometric mean 1.137
* All benchmarks have **the same** allocations


The following thresholds were used for comparing the benchmark speeds:
* Mann–Whitney U test with statistical test for significance of **5%**
* Only results indicating a difference greater than **10%** and **0.3 ns** are considered.

Allocation changes below **0.5%** are ignored.


### Benchmark details


<details><summary><strong>Benchmarks.Trace.AgentWriterBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`WriteAndFlushEnrichedTraces`|net472|942μs|4.5μs|18μs|0| 0|0|3.16&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|755μs|4.16μs|26.3μs|0| 0|0|2.57&nbsp;KB|
|#2347|`WriteAndFlushEnrichedTraces`|net472|972μs|6.34μs|61.8μs|0| 0|0|3.16&nbsp;KB|
|#2347|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|721μs|2.32μs|8.02μs|0| 0|0|2.57&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AspNetCoreBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`SendRequest`|netcoreapp3.1|352μs|1.05μs|3.92μs|0.176| 0|0|19.67&nbsp;KB|
|#2347|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|#2347|`SendRequest`|netcoreapp3.1|349μs|916ns|3.43μs|0.174| 0|0|19.67&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.DbCommandBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`ExecuteNonQuery`|net472|1.8μs|6.59ns|25.5ns|0.0924| 0.000915|0|586&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`ExecuteNonQuery`|netcoreapp3.1|1.61μs|9.27ns|73ns|0.00863| 0|0|624&nbsp;B|
|#2347|`ExecuteNonQuery`|net472|1.78μs|7.31ns|28.3ns|0.0926| 0.000882|0|586&nbsp;B|
|#2347|`ExecuteNonQuery`|netcoreapp3.1|1.58μs|5.46ns|21.1ns|0.00869| 0|0|624&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.ElasticsearchBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`CallElasticsearch`|net472|2.81μs|16.3ns|128ns|0.124| 0|0|794&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`CallElasticsearch`|netcoreapp3.1|1.72μs|4.38ns|16.4ns|0.0107| 0|0|784&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`CallElasticsearchAsync`|net472|2.93μs|9.98ns|36ns|0.145| 0|0|931&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`CallElasticsearchAsync`|netcoreapp3.1|1.94μs|10.1ns|46.1ns|0.013| 0|0|904&nbsp;B|
|#2347|`CallElasticsearch`|net472|2.51μs|14.1ns|89.4ns|0.124| 0|0|794&nbsp;B|
|#2347|`CallElasticsearch`|netcoreapp3.1|1.75μs|7.77ns|29.1ns|0.0107| 0|0|784&nbsp;B|
|#2347|`CallElasticsearchAsync`|net472|3.26μs|13.4ns|51.8ns|0.144| 0|0|931&nbsp;B|
|#2347|`CallElasticsearchAsync`|netcoreapp3.1|1.89μs|4.44ns|16ns|0.0124| 0|0|904&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.GraphQLBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`ExecuteAsync`|net472|2.97μs|14.4ns|66.1ns|0.164| 0|0|1.05&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`ExecuteAsync`|netcoreapp3.1|2.13μs|9.44ns|36.6ns|0.0147| 0|0|1.02&nbsp;KB|
|#2347|`ExecuteAsync`|net472|2.96μs|16.7ns|118ns|0.164| 0|0|1.05&nbsp;KB|
|#2347|`ExecuteAsync`|netcoreapp3.1|2.18μs|11.3ns|54.2ns|0.0135| 0|0|1.02&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.HttpClientBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`SendAsync`|net472|8.02μs|21.9ns|82.1ns|0.349| 0|0|2.26&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`SendAsync`|netcoreapp3.1|4.58μs|22.6ns|98.4ns|0.0307| 0|0|2.16&nbsp;KB|
|#2347|`SendAsync`|net472|7.8μs|17.6ns|63.6ns|0.352| 0|0|2.26&nbsp;KB|
|#2347|`SendAsync`|netcoreapp3.1|4.63μs|24.9ns|169ns|0.0294| 0|0|2.16&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.ILoggerBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|net472|3.69μs|14.6ns|54.8ns|0.225| 0|0|1.44&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|netcoreapp3.1|3.39μs|12.1ns|47ns|0.0203| 0|0|1.52&nbsp;KB|
|#2347|`EnrichedLog`|net472|3.74μs|14.9ns|55.9ns|0.225| 0|0|1.44&nbsp;KB|
|#2347|`EnrichedLog`|netcoreapp3.1|3.45μs|18ns|95.5ns|0.0205| 0|0|1.52&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.Log4netBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|net472|321μs|1.8μs|12.6μs|0.31| 0.155|0|4.31&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|netcoreapp3.1|268μs|1.39μs|6.82μs|0| 0|0|4.19&nbsp;KB|
|#2347|`EnrichedLog`|net472|311μs|1.08μs|4.18μs|0.314| 0.157|0|4.31&nbsp;KB|
|#2347|`EnrichedLog`|netcoreapp3.1|261μs|1.16μs|4.36μs|0| 0|0|4.19&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.NLogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|net472|8.49μs|39.1ns|152ns|0.502| 0|0|3.21&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|netcoreapp3.1|7.12μs|39.3ns|264ns|0.0515| 0|0|3.58&nbsp;KB|
|#2347|`EnrichedLog`|net472|8.52μs|26.3ns|102ns|0.5| 0|0|3.21&nbsp;KB|
|#2347|`EnrichedLog`|netcoreapp3.1|6.85μs|21.5ns|83.3ns|0.0504| 0|0|3.58&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.RedisBenchmark</strong> - Faster :tada: Same allocations :heavy_check_mark:



#### Faster :tada: in #2347

|  Benchmark         | base/diff | Base Median (ns) | Diff Median (ns) | Modality   |
|:----------|-----------:|-----------:|--------:|--------:|
| Benchmarks.Trace.RedisBenchmark.SendReceive&#8209;net472 | 1.137 | 2,708.71 | 2,381.83 | 

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`SendReceive`|net472|2.7μs|13.9ns|68.3ns|0.156| 0|0|1&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`SendReceive`|netcoreapp3.1|2.17μs|7.08ns|27.4ns|0.0138| 0|0|1&nbsp;KB|
|#2347|`SendReceive`|net472|2.39μs|9.44ns|35.3ns|0.157| 0|0|1&nbsp;KB|
|#2347|`SendReceive`|netcoreapp3.1|2.1μs|10.1ns|40.4ns|0.0147| 0|0|1&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SerilogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|net472|6.37μs|23.6ns|85.2ns|0.289| 0|0|1.85&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`EnrichedLog`|netcoreapp3.1|5.76μs|22.7ns|87.7ns|0.02| 0|0|1.47&nbsp;KB|
|#2347|`EnrichedLog`|net472|6.51μs|35.5ns|198ns|0.287| 0|0|1.85&nbsp;KB|
|#2347|`EnrichedLog`|netcoreapp3.1|6.42μs|26.5ns|95.5ns|0.019| 0|0|1.47&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SpanBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`StartFinishSpan`|net472|1.02μs|5.28ns|26.4ns|0.0704| 0|0|449&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`StartFinishSpan`|netcoreapp3.1|990ns|3.25ns|12.2ns|0.00595| 0|0|448&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`StartFinishScope`|net472|1.22μs|5.96ns|24.6ns|0.083| 0|0|530&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/233e6c5e49ab8cbf226e40999b88e32424cb6022)|`StartFinishScope`|netcoreapp3.1|1.17μs|6.83ns|63.4ns|0.00771| 0|0|568&nbsp;B|
|#2347|`StartFinishSpan`|net472|1.03μs|5.76ns|46.8ns|0.0704| 0|0|449&nbsp;B|
|#2347|`StartFinishSpan`|netcoreapp3.1|970ns|2.98ns|11.5ns|0.0063| 0|0|448&nbsp;B|
|#2347|`StartFinishScope`|net472|1.21μs|4.85ns|18.8ns|0.0827| 0|0|530&nbsp;B|
|#2347|`StartFinishScope`|netcoreapp3.1|1.16μs|6.05ns|31.4ns|0.00791| 0|0|568&nbsp;B|
</details>

**Review by andrewlock — APPROVED** [review]

**Comment by andrewlock** [comment]
## Code Coverage Report :bar_chart:

:heavy_check_mark: Merging #2347 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/872a15956cf369773d48b25c7eab644b08d74701) will **not change** line coverage
:heavy_check_mark: Merging #2347 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/872a15956cf369773d48b25c7eab644b08d74701) will **not change** branch coverage
:heavy_check_mark: Merging #2347 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/872a15956cf369773d48b25c7eab644b08d74701) will will **decrease** complexity by `20`


|           | [master](https://github.com/DataDog/dd-trace-dotnet/tree/872a15956cf369773d48b25c7eab644b08d74701) | #2347       | Change   | 
|:----------|:-----------:|:-----------:|:--------:|
| Lines     | `10429` / `14678` | `10446` / `14673` |          |
| Lines %   | `71%`      | `71%`      |  `0%` :heavy_check_mark:  |
| Branches  | `5361` / `7739` | `5353` / `7719` |          |
| Branches %| `69%`      | `69%`      |  `0%` :heavy_check_mark:  |
| Complexity|   `8850`      | `8830`        |  `-20`  :heavy_check_mark:    |

View the full report for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=88012&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/88012/artifacts?artifactName=Code%20Coverage%20Report_88012&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2347](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=88005&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/88005/artifacts?artifactName=Code%20Coverage%20Report_88005&api-version=6.0&%24format=zip)

### Datadog.Trace Breakdown :heavy_check_mark:

|        | [master](https://github.com/DataDog/dd-trace-dotnet/tree/872a15956cf369773d48b25c7eab644b08d74701) | #2347       | Change   | 
|:-------|:-----------:|:-----------:|:--------:|
| Lines %| `71%`      | `71%`       |  `0%` :heavy_check_mark:  |
| Branches %| `69%`      | `69%`       |  `0%` :heavy_check_mark:  |
| Complexity| `8850`      | `8830`       |  `-20` :heavy_check_mark:  |

The following classes have significant coverage changes.

| File    | Line coverage change | Branch coverage change | Complexity change |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Couchbase.IIOServiceExecuteIntegrationBis](https://github.com/DataDog/dd-trace-dotnet/tree/19c8084826d5edca883d933af37a14c14402fb37/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Couchbase/IIOServiceExecuteIntegrationBis.cs) | `-100%` :no_entry: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2.TestMethodRunnerExecuteIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/19c8084826d5edca883d933af37a14c14402fb37/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestMethodRunnerExecuteIntegration.cs) | `-33%` :no_entry: | `-19%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit.NUnitCompositeWorkItemSkipChildrenIntegration](https://github.com/DataDog/dd-trace-dotnet/tree/19c8084826d5edca883d933af37a14c14402fb37/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/NUnitCompositeWorkItemSkipChildrenIntegration.cs) | `-8%` :no_entry: | `-8%` :no_entry:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Agent.StreamFactories.UnixDomainSocketStreamFactory](https://github.com/DataDog/dd-trace-dotnet/tree/19c8084826d5edca883d933af37a14c14402fb37/tracer/src/Datadog.Trace/Agent/StreamFactories/UnixDomainSocketStreamFactory.cs) | `8%` :heavy_check_mark: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |
| [Datadog.Trace.Agent.Transports.HttpStreamRequestFactory](https://github.com/DataDog/dd-trace-dotnet/tree/19c8084826d5edca883d933af37a14c14402fb37/tracer/src/Datadog.Trace/Agent/Transports/HttpStreamRequestFactory.cs) | `20%` :heavy_check_mark: | `0%` :heavy_check_mark:   | `0` :heavy_check_mark: |


The following classes were added in #2347:

| File    | Line coverage | Branch coverage | Complexity |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.TracingProcessManager](https://github.com/DataDog/dd-trace-dotnet/pull/2347/files) | `100%` | `100%` | `27` |


1 classes were removed from Datadog.Trace in #2347


View the full reports for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=88012&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/88012/artifacts?artifactName=Code%20Coverage%20Report_88012&api-version=6.0&%24format=zip)  
* [HTML report for this PR #2347](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=88005&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/88005/artifacts?artifactName=Code%20Coverage%20Report_88005&api-version=6.0&%24format=zip)

**Review by lucaspimentel — APPROVED** [review]

**Pull request merged**